package.json

```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "c8": "7.7.1",
+    "escodegen-wallaby": "1.6.32",
```

Comment:
I added this as a dev dependency >.<, but its just for running the test.  It just seems like this library is the easiest way to convert an ASTNode back into the actual code, without doing specific AST node type checks like `if (node.type === 'ObjectExpression')...`